cmd/ctr/commands/images/import.go

```diff
@@ -34,45 +35,58 @@ var importCommand = cli.Command{
 	ArgsUsage: "[flags] <in>",
 	Description: `Import images from a tar stream.
 Implemented formats:
-- oci.v1     (default)
+- oci.v1
+- docker.v1.1
+- docker.v1.2
 
 
-For oci.v1 format, you need to specify --oci-name because an OCI archive contains image refs (tags)
-but does not contain the base image name.
+For OCI v1, you may need to specify --base-name because an OCI archive
+contains only partial image references (tags without the base image name).
```

Comment:
likely/recommended to contain only tags, but IIUC it is also valid to include the base image name?

Reply:
And we should have tests (can be a separate PR)

Reply:
Updated to say it may contain it, I think we should be exporting with the base name otherwise there is important information which gets excluded.